projects/public-search/src/app/patron-profile/patron-profile-personal/patron-profile-personal.component.html

```diff
@@ -0,0 +1,103 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2021 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="record">
+  <div id="personal-section" class="row px-3 mt-2 my-1">
+    <div class="col mr-1 d-none d-lg-block">
+      <div class="row p-2 bg-dark rounded text-light">
+        <div class="col-lg-12 pl-0" translate>Personal details</div>
+      </div>
+    </div>
+  </div>
+
+  <div id="personal-data" class="row px-3 mt-2">
+    <div class="col-12">
```

Comment:
remove and ensure than all child that you would view on a single line are "module 12"

Reply:
done